apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts

```diff
@@ -229,6 +215,7 @@ export class UpsertWorkflowUseCase {
       description: workflowDto.description,
       tags: workflowDto.tags,
       active: workflowDto.active ?? true,
+      identifier: workflowDto.identifier,
```

Comment:
`workflowId` or `identifier`?

Reply:
taking into to consideration the new naming it is workflowId, i will update it.